assets/admin/js/bb-analysis.js

```diff
@@ -0,0 +1,20 @@
+(function($){
+	var ClassicSEOIntegration = function() {
+			this.hooks()
+		}
+
+		ClassicSEOIntegration.prototype.hooks = function() {
+			classicSEOApp.registerPlugin( 'bb-seo' )
+			wp.hooks.addFilter( 'cpseo_content', 'bb-seo', function(content) {
+				return window.classicSEO.beaverbuilder.pagedata;
```

Comment:
It looks like `window.classicSEO.beaverbuilder` will be undefined if Beaver Builder is not loaded. Maybe this is not actually a problem though, since the script will only be registered if BB is loaded...

Reply:
Yes, the script won't load in BB is not installed and active.